api/src/paths/artifact/security/requestAccess.ts

```diff
@@ -0,0 +1,157 @@
+import { RequestHandler } from 'express';
+import { Operation } from 'express-openapi';
+import { getAPIUserDBConnection, getDBConnection } from '../../../database/db';
+import { GCNotifyService } from '../../../services/gcnotify-service';
+import { getLogger } from '../../../utils/logger';
+
+const defaultLog = getLogger('/api/artifact/security/requestAccess');
+
+export const POST: Operation = [requestAccess()];
+
+POST.apiDoc = {
+  description: 'Request access to secure artifacts in Biohub.',
+  tags: ['documents', 'security', 'biohub'],
+  security: [
+    {
+      OptionalBearer: []
```

Comment:
(lets double check) but I think we do want to enforce that the user must be logged in before they can request access? Currently we don't track anything, but we will eventually probably need to track the request against a system user with valid IDIR or BCeID.

Reply:
Per discussion with Jer, according to stakeholders, users do not need to be signed in to send a request.